docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -248,30 +248,31 @@ The next step in testing your new module is to consume it with an Ansible playbo
 Testing basics
 ====================
 
-These two examples will get you started with testing your module code. Please review our :ref:`testing <developing_testing>` section for more detailed
+These two examples will get you started with testing your module code. The :ref:`testing <developing_testing>` section has more detailed
 information, including instructions for :ref:`testing module documentation <testing_module_documentation>`, adding :ref:`integration tests <testing_integration>`, and more.
 
 Sanity tests
 ------------
 
-You can run through Ansible's sanity checks in a container:
+You can run through the Ansible sanity checks in a container. By default the test command will run tests on all supported versions of Python. To limit your test run to a specific Python version:
 
 ``$ ansible-test sanity -v --docker --python 2.7 MODULE_NAME``
```

Comment:
Maybe it's worth to add `default` after `--docker`. This is relevant when you remove the `--python 2.7`: suddenly `MODULE_NAME` will be interpreted as the docker container to use.
```suggestion
``$ ansible-test sanity -v --docker default --python 2.7 MODULE_NAME``
```